src/components/views/Page.tsx

```diff
@@ -22,14 +22,29 @@ const PageView = ({ children, title, breadcrumbs, actions }: PageViewProps) => {
           <div className="flex gap-1">
             {breadcrumbs.map((breadcrumb, index) => (
               <div key={index} className="flex flex-row gap-1">
-                <Link href={breadcrumb.href}>
-                  <Text element="p" as={"smallText"}>
+                {breadcrumb.title === title ? (
```

Comment:
It's not guaranteed that the page's title should match what was entered into the breadcrumb, checking if it's the final index instead is favourable here